hungarian/README.md

```diff
@@ -0,0 +1,52 @@
+Attila is subscribed in Liblouis list. If anybody have question the hungarian braille tables related, feel free ask a quastion in developer community.
+Please ask beginner table maintainers:
+If would like doing bigger changes in hungarian braille tables related, please not commit the changes official the source [Liblouis repository](https://github.com/liblouis/liblouis) until all testcases are not passed, because hungarian language is a very complex language.
+If a maintenance change related have a question, have possibility to write e-mail a question the table maintainer with following addresses:
+* Full official work time used e-mail address, with awailable the hungarian braille tables: hammer.attila at infoalap dot hu
+* Private e-mail address, with Attila subscribed into Liblouis mailing list: hammera at pickup dot hu
+
+## Hungarian language awailable downloadable documents
+The Hungarian Braille Committee official subpage is following:
+[The Hungarian Braille Committee official subpage](https://www.mvgyosz.hu/tudjon-meg-tobbet-hu/braille-iras/)
+Following two documents are standard braille embossers USA braille character set compatible, windows CRLF line terminator wrapped documents, with containing the hungarian braille alphabet. Without changes, this documents are free usable:
+* [Braille Alphabet Book (first part, USA BRF document, formatted with Windows line wrap)](https://www.mvgyosz.hu/wp-content/uploads/2020/04/braille-%C3%A1b%C3%A9c%C3%A9sk%C3%B6nyv01.brf)
+* [Braille Alphabet Book (second part, USA BRF document, formatted with Windows line wrap)](https://www.mvgyosz.hu/wp-content/uploads/2020/04/braille-%C3%A1b%C3%A9c%C3%A9sk%C3%B6nyv02.brf)
+* [Hungarian language used braille dots (an EXCEL table, with containing in 12 worksheet the various braille rules)](https://www.mvgyosz.hu/wp-content/uploads/2020/04/magyar-braille-%C3%ADr%C3%A1s-jelei.xls)
+* [Summary collection of decisions concerning Braille adopted by the presidency of the Hungarian National Association of the Blind and Visually Impaired](https://www.mvgyosz.hu/wp-content/uploads/2020/04/braille-%C3%ADr%C3%A1s-eln%C3%B6ks%C3%A9gi-hat%C3%A1rozatok.docx)
+* [The medicine boxes braille standards collection documents (dot sizes, line length, etc.)](https://www.mvgyosz.hu/wp-content/uploads/2020/04/braille-gy%C3%B3gyszercsomagol%C3%A1sokon.doc)
+## Marburg Medium standard in Hungary
+The Marburg Medium standard specifies the following parameters for the size of Braille dots and their spacing:
+* Diameter of a point: 1.6 mm
+* The distance between the centers of two points inside a letter (horizontally): 2.5 mm
+* Distance between the centers of two points within a letter (vertically): 2.5 mm
+* The distance between the same point of two letters within a word: 6 mm
+* Distance between the same point of two letters if there is a space between the two letters: 12 mm
+* Line spacing (between the same point of two letters): 10 mm
```

Comment:
I'm not sure if this is relevant information?

Reply:
This issue is not resolved yet. I don't think this information is relevant for Liblouis.